ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/disconnected/private/cco-manual-workload-identity/provision/cucushift-installer-rehearse-gcp-ipi-disconnected-private-cco-manual-workload-identity-provision-chain.yaml

```diff
@@ -0,0 +1,28 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-disconnected-private-cco-manual-workload-identity-provision
+  steps: 
+  - ref: openshift-cluster-bot-rbac
+  - ref: gcp-provision-vpc
+  - ref: ignition-bastionhost
+  - ref: gcp-provision-bastionhost
+  - chain: mirror-images-by-oc-adm
+  - ref: proxy-config-generate
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-gcp
+  - ref: ipi-conf-manual-creds
+  - ref: ipi-conf-gcp-oidc-creds-provision
```

Comment:
For this workflow, maybe we can wait for https://github.com/openshift/release/pull/33641/files get merged, then insert `ipi-conf-manual-creds-remove-unnecessary-creds` here too.

Reply:
@jianlinliu Do you think I shall move the updates on step `gather-gcp-console` (which triggered many jobs) to another PR, and then wait for #33641getting merged in this PR?

Reply:
I saw you already updated `gather-gcp-console` step in this PR. Do you mean will insert `ipi-conf-manual-creds-remove-unnecessary-creds` in another PR?

Reply:
I meant to revert the changes on `gather-gcp-console` and wait for #33641 getting merged. But as #33641 had been merged, so please ignore my comment. Just inserted `ipi-conf-manual-creds-remove-unnecessary-creds`.